app/main.py

```diff
@@ -1,6 +1,41 @@
+import os
+
+import requests
+
+URL = "https://api.weatherapi.com/v1/current.json"
+
+API_KEY = os.getenv("API_KEY")
+
+PARAMS = {
+    "key": API_KEY,
+    "q": "Paris",
+}
+
+
+def get_res() -> dict:
+    try:
+        print("Performing request to Weather API to city Paris...")
+        response = requests.get(URL, params=PARAMS)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as err:
+        print(f"HTTP ERROR: {err}")
+    except requests.exceptions.RequestException as err:
+        print(f"REQUEST ERROR: {err}")
+    return response.json()
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    weather = get_res()
```

Comment:
The `get_weather` function does not handle the case where `get_res()` might return `None` or an unexpected structure due to an error in the API request. Consider adding checks to ensure `weather` contains the expected data before accessing it.